src/table/utils/styling-utils.js

```diff
@@ -154,6 +153,26 @@ export function getBodyCellStyle(layout, theme) {
   };
 }
 
+/**
+ * Get style of total cell
+ *
+ * @param {Object} totalHeaderStyle
+ * @param {String} totalsPosition
+ * @returns total cell style regarding the total head position
+ */
+export function getTotalsCellStyle(totalCellStyle, totalsPosition) {
+  totalCellStyle.fontWeight = 'bold';
+  if (totalsPosition === 'top') {
+    totalCellStyle.borderWidth = '0px 1px 2px 0px';
```

Comment:
would be good to cover here as well.

Reply:
wait what I never pushed that... I'll fix that